### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kylelhk**
## Change Summary 📝

Provide a more detailed and structured README

---

## Change Form 📋
Please mark as NA if not applicable. If a certain criterion is not met, kindly provide a reason.

- [ ] The pull request title has an issue number. - N/A
- [ ] I have performed a self-review of my own code. - N/A
- [ ] I have commented my code, particularly in hard-to-understand areas. - N/A
- [x] I have made corresponding changes to the documentation.
- [ ] Relevant tests (if applicable) have been added or updated. - N/A

---

## Any background context you want to provide? 🖼️

N/A

## Reviewers 🕵️

(Optional) Suggest reviewers that might be interested in this PR.

### CODE REVIEW

**Review by WendyW09 — COMMENTED** [review]
It looks good for me, every part is clearly written.

**Comment by kylelhk** [comment]
> It looks good for me, every part is clearly written.

@WendyW09 Please submit a review under the "Files changed" tab.  Thanks.
image

**Comment by ChangwuWu** [comment]
It looks good for me

File: README.md
Comment (yylocky) [inline README.md:1]: Good to go!

**Review by yylocky — APPROVED** [review]

**Pull request merged**